frappe/utils/__init__.py

```diff
@@ -867,30 +860,6 @@ def call(fn, *args, **kwargs):
 	return json.loads(frappe.as_json(frappe.call(fn, *args, **kwargs)))
 
 
-# Following methods are aken as-is from Python 3 codebase
-# since gzip.compress and gzip.decompress are not available in Python 2.7
-def gzip_compress(data, compresslevel=9):
```

Comment:
We shouldn't remove functions in utils. You can deprecate this with @depracate decorator instead and we can remove it at later date (after few months)

Reply:
https://gameplan.frappe.cloud/g/engineering/projects/29/discussion/2197/policy-on-breaking-changes-and-branching